src/internet_identity/tests/integration/http.rs

```diff
@@ -17,7 +17,6 @@ fn ii_canister_serves_http_assets() -> Result<(), CallError> {
     let assets: Vec<(&str, Option<&str>)> = vec![
         ("/", None),
         ("/about", None),
-        ("/index.html", None),
         ("/index.js", Some("gzip")),
         ("/index.css", None),
         ("/loader.webp", None),
```

Comment:
I think we should have some test that verifies that assets are served and also certified correctly. This is the only test that would fail if we were to screw up asset certification. So what about we don't make this a full list of assets but rather a smaller selection of assets as a sanity check.
I.e.:
* `/` because we want to have some asset there anyway
* `/index.js` because it is gzipped
* `/.well-known/ic-domains` because it is the only asset that does _not_ have a file in `dist` and it would break the new domain

WDYT?

Reply:
SGTM! Good thinking.